docs/features/completion.rst

```diff
@@ -31,3 +31,50 @@ similar to the following to your class which inherits from ``cmd2.Cmd``::
 
     # Make sure you have an "import functools" somewhere at the top
     complete_bar = functools.partialmethod(cmd2.Cmd.path_complete, path_filter=os.path.isdir)
+
+
+Tab Completion Using Argparse Decorators
```

Comment:
This would be a good place to include most, if not all, of the header to [argparse_custom.py](https://github.com/python-cmd2/cmd2/blob/master/cmd2/argparse_custom.py).

The documentation there is pretty thorough. What do you think?

Reply:
I think we should include that either via an automodule or autoclass Sphinx Autodoc

Reply:
Unfortunately Sphinx autodoc is [very particular](http://www.sphinx-doc.org/en/master/usage/extensions/autodoc.html) about formatting of docstrings for it to work.  

For this to work, the docstrings must be written in correct reStructuredText. You can then use all of the usual Sphinx markup in the docstrings, and it will end up correctly in the Sphinx documentation.

So to use ``automodule`` to import this documentation would require quite a bit of reformatting of that top-of-module docstring.

Reply:
@kmvanbrunt If you want to either fix the docstring so it works with **automodule** or manually put the equivalent documentation in the reStructuredText, feel free.  I don't have time to do this right now.  Alternatively, if you like we could create a new issue to do it so we don't forget about it.

Reply:
I created #800 to track this work